docs/en-US/component/upload.md

```diff
@@ -89,53 +89,106 @@ upload/manual
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| action ^(required)            | request URL.                                                                                                                                                                          | ^[string]                                                                                                                        | #                                                                                                                  |
+| headers                       | request headers.                                                                                                                                                                      | ^[object]`Headers \| Record<string, any>`                                                                                        | —                                                                                                                  |
+| method                        | set upload request method.                                                                                                                                                            | ^[string]                                                                                                                        | post                                                                                                               |
+| multiple                      | whether uploading multiple files is permitted.                                                                                                                                        | ^[boolean]                                                                                                                       | false                                                                                                              |
+| data                          | additions options of request.                                                                                                                                                         | ^[object]`Record<string, any> \| Awaitable<Record<string, any>> \| ((rawFile: UploadRawFile) => Awaitable<Record<string, any>>)` | () => Mutable<{}>                                                                                                  |
```

Comment:
```suggestion
| data                          | additions options of request. support `Awaitable` data and `Function` since v2.3.13 | ^[object]`Record<string, any> \| Awaitable<Record<string, any>>` / ^[Function]`((rawFile: UploadRawFile) => Awaitable<Record<string, any>>)` | {}                                                                                                  |
```

Some details are missing here

Reply:
updated.